File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {

        vector<int> nums3;

        int i = 0;
        int j = 0;

        while(i < nums1.size() && j < nums2.size()){

            if(nums1[i] <= nums2[j]){
                nums3.push_back(nums1[i]);
                i++;
            }
            else{
                nums3.push_back(nums2[j]);
                j++;
            }
        }

        while(i < nums1.size()){
            nums3.push_back(nums1[i]);
            i++;
        }

        while(j < nums2.size()){
            nums3.push_back(nums2[j]);
            j++;
        }

        int n = nums3.size();

        if(n % 2 == 0){
            return (nums3[n/2 - 1] + nums3[n/2]) / 2.0;
        }

        return nums3[n/2];
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        nums3=nums1+nums2
        nums3.sort()
        l=len(nums3)
        if l%2==1:
            median=nums3[l//2]
        else:
            median=(nums3[l//2]+nums3[l//2-1])/2.0
        return median    